mentors/README.md

```diff
@@ -1,78 +1,140 @@
+2. **Find a project idea**: Either come up with a new mentorship project idea or find an existing mentorship project idea that you would like to mentor for.
+   The definition of a good project idea varies from program to program. Different mentorship programs and project initiatives have their own unique focuses and areas of emphasis. For instance, some projects place a greater emphasis on coding and software development, while others prioritise documentation and technical writing. The specific goals and objectives of each program may vary, but generally, they strive to provide valuable learning experiences and support to participants in their respective fields.
+3. **Submit your project idea**: Submit your project idea to the [CNCF mentoring repository](https://github.com/cncf/mentoring). You can use the [project idea template](https://github.com/cncf/mentoring/blob/main/PROJECT_IDEA_TEMPLATE.md). Information on how to submit your project idea will be provided in the program announcement that will be sent out.
+4. **Review mentee profiles and proposals**: Once the mentee profiles and proposals are submitted for your project idea, you will  have the opportunity to review them. Program administrators will provide you with more information on how to review the proposals and the main criteria for acceptance.
+5. **Mentor the mentees**: Once the mentees are accepted into the program, you will be able to start mentoring them!
 
-## Success Stories
+While the processes may vary from program to program, you will receive detailed instructions from the program administrators at each step, including a timeline. This ensures that you are well informed about what needs to be done and how to proceed throughout the program!
+
+## Programs
 
+For the up-to-date list of programs, please check the [CNCF mentoring repository](https://github.com/cncf/mentoring).
 
+## Example mentoring timeline
+
+* GSoC at CNCF 2023 is [announced](https://github.com/cncf/mentoring/discussions/848)
+  * Mentors send their project idea PRs ([example](https://github.com/cncf/mentoring/pull/810))
+* Mentee applications start ([announcement](https://github.com/cncf/mentoring/discussions/892))
+  * Mentee candidates interact with the community and the mentors
+  * Mentee candidates submit their proposals to the GSoC platform
+*  Mentee applications close ([announcement](https://github.com/cncf/mentoring/discussions/918))
+  * Mentors review the proposals and provide feedback
+* Accepted mentees are [announced](https://github.com/cncf/mentoring/discussions/954)
+  * Mentees bond with the community and the mentors
+  * Mentors and mentees start working on their projects
+* TBA: (we're in the middle of the program)
+
+## Success Stories
 
-## FAQs
+TBD
 
```

Comment:
Created https://github.com/cncf/mentoring/issues/1025 for this draft section. I would like to merge this PR regardless.